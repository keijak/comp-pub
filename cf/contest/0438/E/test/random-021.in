163 102
559
454
577
116
521
738
686
351
414
591
383
959
923
851
208
248
340
642
720
943
629
81
888
995
894
517
765
529
176
400
363
402
439
43
374
769
333
553
14
613
266
973
616
221
461
694
904
716
892
936
612
582
435
3
908
319
587
680
917
619
499
961
937
708
589
184
83
942
916
838
228
134
643
357
412
449
331
79
177
91
431
912
263
905
971
84
202
506
919
124
572
168
948
6
385
808
238
356
246
268
768
237
19
896
809
960
836
338
27
889
929
502
145
618
876
729
381
480
445
289
26
810
133
303
309
432
532
551
40
443
831
979
39
997
601
903
630
49
639
664
390
378
159
996
421
485
193
872
493
491
661
90
962
947
538
62
545
38
845
683
447
953
349
