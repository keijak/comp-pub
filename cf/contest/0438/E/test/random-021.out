0
0
1
0
0
3
0
0
9
0
0
31
0
1
113
0
4
431
1
19
1697
4
86
6847
19
394
28162
88
1804
117635
408
8298
497688
1899
38304
2128243
8867
177409
9184081
41547
824107
39943848
195237
3838174
174912671
919894
17917135
770527667
4344341
83811371
417602604
20559094
392763845
209474800
97470598
845375949
960837026
462853519
680703847
836429288
204595523
865570982
44203871
498163211
599697995
878042179
49268889
699666298
864171589
832814493
624989443
12704316
775142108
326062788
313046527
459290328
818643209
514007128
230118892
452643204
321066708
459178362
972683577
71808664
418821268
774008081
583753540
766895406
703197567
770114915
887267656
225208337
526057002
433918579
200531290
796070745
984670220
725297495
439805277
843584595
100322687
190078712
